tensorboard/plugins/mesh/BUILD

```diff
@@ -167,6 +195,20 @@ py_binary(
     ],
 )
 
+py_binary(
+    name = "mesh_demo_v2",
+    srcs = ["mesh_demo_v2.py"],
+    srcs_version = "PY2AND3",
+    visibility = [
+        "//visibility:public",
+    ],
+    deps = [
+        ":demo_utils",
+        ":summary_v2",
+        "//tensorboard/compat:tensorflow",
```

Comment:
This should also get placeholder deps for numpy and for absl app and flags (they're all `expect_xx_installed` targets).